package.json

```diff
@@ -1,19 +1,26 @@
 {
-  "name": "blackninjas-backend",
+  "name": "the-snipers-backend",
   "version": "1.0.0",
   "description": "A platform to make company global travel and accommodation easy and convenient for the strong workforce of savvy members of staff, by leveraging the modern web.",
   "main": "index.js",
   "scripts": {
-    "test": "echo \"Error: no test specified\" && exit 1"
+    "test": "NODE_ENV=test nyc --reporter=text --reporter=html ./node_modules/.bin/_mocha ./src/test/**.test.js ",
+    "coveralls": "NODE_ENV=test nyc report --reporter=text-lcov | coveralls"
```

Comment:
Why are you setting up coveralls in this MR? The scope of this MR should be to create a detailed readme file

Reply:
I think that's because I haven't merged the previous PR on coveralls. I had created a new branch for this but when I went to create PR I saw that it was all here while I raised a PR before

Reply:
Removed in the latest commit